package/rules/typescript.js

```diff
@@ -0,0 +1,21 @@
+const { join, resolve } = require('path')
+const { cache_path: cachePath, source_path: sourcePath, resolved_paths: resolvedPaths } = require('../config')
+const { nodeEnv } = require('../env')
+
+// Process application TypeScript code with Babel.
+// Uses application .babelrc to apply any transformations
+module.exports = {
+  test: /\.(ts|tsx)?$/,
```

Comment:
@gopeter Can we use the same babel loader to enable typescript as well?  
https://github.com/rails/webpacker/blob/master/package/rules/babel.js#L8

Reply:
This would be possible if we add the ts/tsx extensions to it. But then Babel would be able to parse Typescript files, even without running `webpacker:install:typescript` before – which is necessary to get the configuration files (like tsconfig.json). I think it would be better if we don‘t mix it up. 

But: we may could add the ts/tsx extensions to the current Babel config in the Typescript install script. Then we could also remove the `environment.loaders.delete('typescript')` step, which you mentioned below.

Reply:
I see, I think that's fine. 

It wouldn't do anything for people who aren't using typescript but will probably fail if they are using typescript and don't have tsconfig.json? But perhaps not, since some of default options would on in the preset itself, right?

Reply:
Could you please test by including extensions as default?